source/common/http/filter/BUILD

```diff
@@ -37,10 +36,10 @@ envoy_cc_library(
     repository = "@envoy",
     deps = [
         "//source/common/config:nats_streaming_well_known_names_lib",
-        "@envoy//source/exe:envoy_common_lib",
         "@solo_envoy_common//source/common/config:solo_metadata_lib",
         "@solo_envoy_common//source/common/http:functional_stream_decoder_base_lib",
         "@solo_envoy_common//source/common/http:solo_filter_utility_lib",
+        "@envoy//source/common/config:metadata_lib",
```

Comment:
Nit:
The `deps` do not seem to be sorted alphabetically. Did you run the formatting script?

Reply:
i did now!